app/components/article-meta.js

```diff
@@ -0,0 +1,15 @@
+import Component from '@ember/component';
+import { inject as service } from '@ember/service';
+
+export default Component.extend({
+  session: service(),
+  /**
+   * Method executed when "favorite" button is clicked.
+   */
+  onFavoriteArticle() {},
```

Comment:
Is there a reason for these empty functions?

Reply:
I have these no-ops as placeholders. I can remove them if you'd like.

Reply:
I decided to just remove them and add code comments for them.